.gitignore

```diff
@@ -1,4 +1,4 @@
 phpunit.xml
 vendor/
 composer.lock
-
+.idea
```

Comment:
this should be removed. It is not a rule related to the project structure, but to your development environment, and so should be handled by your dev environment (either through a global rule or a per-project local rule): https://help.github.com/articles/ignoring-files/#create-a-global-gitignore